ethereumj-core/src/main/java/org/ethereum/jsontestsuite/JSONReader.java

```diff
@@ -71,6 +73,36 @@ public static String getFromUrl(String urlToRead) {
         return result;
     }
 
+    public static String getTestBlobForTreeSha(String shacommit, String testcase){
+
+        String result = getFromUrl("https://api.github.com/repos/ethereum/tests/git/trees/" + shacommit);
+
+        JSONParser parser = new JSONParser();
+        JSONObject testSuiteObj = null;
+
+        List<String> fileNames = new ArrayList<String>();
+        try {
+            testSuiteObj = (JSONObject) parser.parse(result);
+            JSONArray tree = (JSONArray)testSuiteObj.get("tree");
+
+            for (Object oEntry : tree) {
+                JSONObject entry = (JSONObject) oEntry;
+                String testName = (String) entry.get("path");
+                if ( testName.equals(testcase) ) {
+                    String blobresult = getFromUrl( (String) entry.get("url") );
+
+                    testSuiteObj = (JSONObject) parser.parse(blobresult);
+                    String blob  = (String) testSuiteObj.get("content");
+                    byte[] valueDecoded= Base64.decodeBase64(blob.getBytes() );
+                    //System.out.println("Decoded value is " + new String(valueDecoded));
```

Comment:
question: is it include also the files that are no changed in that
particular commit ?